SparkleLayers/GreenLayer.pde

```diff
@@ -0,0 +1,18 @@
+public class CloudLayer extends Layer {
```

Comment:
let's name this green layer (not sure how these got messed up with the names of the files heh)